cyral/assets/configuration/spec.yaml

```
@@ -0,0 +1,19 @@
+name: Cyral
+files:
+- name: cyral.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/openmetrics
+       overrides:
+           prometheus_url.example: http://localhost:9018/metrics
```

```suggestion
       overrides:
         prometheus_url.example: http://localhost:9018/metrics
```

My suggestion left a bit too many indents, this should fix. To sync the `conf.yaml.example` file, you can run `ddev validate config --sync`

---

Oh, thank you!